student.rb

```diff
@@ -0,0 +1,14 @@
+require relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom:, id: rand(1..1000), name: 'Unknown', age: nil, parent_permission: true)
+    super(id, name, age, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Please be extra careful when you're using named parameters like `parent_permission`. If you look closely, your super function, in this case, `Person` expects the `parent_permission` parameter as a key: value. So the correct way to send the `parent_permission` in your call to super() would be:

```suggestion
  def initialize(classroom:, id: rand(1..1000), name: 'Unknown', age: nil, parent_permission: true)
    super(id, name, age, parent_permission: parent_permission)
    @classroom = classroom
  end
```

Please consider applying this change! 🤓 🔧